### PR TITLE
[full-ci] Move language selection to the account page and kill settings app

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
The language selection has been moved from the settings app to the personal account page. The settings app has been removed from the default configs because we don't need it currently.

Fixes https://github.com/owncloud/web/issues/8290

Needs a follow-up PR in oCIS to adjust the configs there.

## Screenshots (if appropriate):
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

**Review by kulmann — CHANGES_REQUESTED** [review]
Found two issues:
- the `Edit` button in the top right of the account page can be removed for ocis... there is no external editing anymore now with this PR
- `listValues` returns `null` if no language has been set before. Your implementation doesn't handle that, I don't see a rendered dropdown and a console error instead.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/31978/53/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIAccount-accountInformation_feature-L10.png
  


</p></details>
:boom: The oCISBasic tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/31981/13/1
<details><summary>:boom: The acceptance tests failed on retry. Please find the screenshots inside ...</summary>

<p>


### webUIAccount-accountInformation_feature-L10.png
  


</p></details>
:boom: The oC10Basic tests pipeline failed. The build has been cancelled.

**Review by dschmidt — APPROVED** [review]

File: packages/web-integration-oc10/lib/Controller/ConfigController.php [hunk 1]
```diff
@@ -124,4 +125,20 @@ private function addOC10AppsToConfig(array $config): array {
...
+    /**
+     * Add an account edit link to the config.
+     *
+     * @param array $config
+     * @return array
+     */
+    private function addAccountEditLinkToConfig(array $config): array {
+        $options = $config['options'] ?? [];
+        $serverUrl = $this->request->getServerProtocol() . '://' . $this->request->getServerHost();
+        $options['accountEditLink'] = [
```
Comment (kulmann) [inline packages/web-integration-oc10/lib/Controller/ConfigController.php:138]: This should only be done if the `config.json` loaded from disk didn't specify it on its own. I.e. if `acountEditLink` already exists you can return early.

**Review by kulmann — APPROVED** [review]

File: packages/web-integration-oc10/lib/Controller/ConfigController.php (hunk 1 above)
Comment (kulmann) [reply packages/web-integration-oc10/lib/Controller/ConfigController.php:141]: see https://github.com/owncloud/web/pull/8294/commits/96363f669b22a3a55a9609cf0fc321458656cd92

**Pull request merged**